src/lib/y2storage/proposal/skip_list_rule.rb

```diff
@@ -0,0 +1,167 @@
+# encoding: utf-8
+#
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/proposal/skip_list_value"
+
+module Y2Storage
+  module Proposal
+    # AutoYaST device skip rule
+    #
+    # @example Using a rule
+    #   Disk = Struct.new(:size_k)
+    #   disk = Disk.new(8192)
+    #   rule = Proposal::SkipListRule.new(:size_k, :less_than, 16384)
+    #   rule.matches?(disk) #=> true
+    #
+    # @example Creating a rule from an AutoYaST profile hash
+    #   hash = { "skip_key" => "size_k", "skip_if_less_than" => 16384,
+    #     "skip_value" => 1024 }
+    #   Proposal::SkipListRule.from_profile_hash(hash)
+    #
+    class SkipListRule
```

Comment:
why it is `SkipListRule` and not simple `SkipRule` I do not see here any reason why it cannot be used outside of SkipList